src/tests/gfx_gltf.edn

```diff
@@ -0,0 +1,34 @@
+(def timestep (/ 1.0 120.0))
+(def Root (Node))
+
+(defn get-view-transform [time]
+  (->
+   (Float4 0 0.2 0.2 1.0) >= .arm-pos
+   time (Math.AxisAngleY) (Math.Rotation) (Math.MatMul .arm-pos) (ToFloat3) >= .arm-pos-3
+   {:Position .arm-pos-3 :Target (Float3 0 0.025 0)} (Math.LookAt)))
```

Comment:
Could use `(defblocks get-view-transform [time])` to avoid the indentation with `(->)`.

Is it completely equivalent though. @sinkingsugar I seem to remember that you told me that since it uses "chainification" there might be some corner cases or performance difference?

Reply:
well `defblocks` is a macro so it's the same thing exactly I think.